<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsDlg</class>
 <widget class="QDialog" name="OptionsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>254</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Global HotKeys</string>
   </property>
   <widget class="QCheckBox" name="chkCopy">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Copy and read</string>
    </property>
   </widget>
   <widget class="QKeySequenceEdit" name="kseCopy">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QKeySequenceEdit" name="kseNow">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QKeySequenceEdit" name="ksePrev">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QKeySequenceEdit" name="kseNext">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QKeySequenceEdit" name="kseStop">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QKeySequenceEdit" name="ksePause">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="clrCopy">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QToolButton" name="clrNow">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QToolButton" name="crlPause">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QToolButton" name="clrNext">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QToolButton" name="clrPrev">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>140</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QToolButton" name="clrStop">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>170</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkNow">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Copy and read later</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkPause">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause / Resume</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkNext">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Read next</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkPrev">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Read Previouse</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkStop">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop reading</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkAdd">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop reading</string>
    </property>
   </widget>
   <widget class="QKeySequenceEdit" name="kseAdd">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="clrAdd">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>191</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Other</string>
   </property>
   <widget class="QCheckBox" name="chkBeep">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Beep when grabbed</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkSysTray">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Minimize to SysTray</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Section Delay</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editDelay">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ms</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>OptionsDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>OptionsDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clrCopy</sender>
   <signal>clicked()</signal>
   <receiver>kseCopy</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clrNow</sender>
   <signal>clicked()</signal>
   <receiver>kseNow</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>crlPause</sender>
   <signal>clicked()</signal>
   <receiver>ksePause</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clrNext</sender>
   <signal>clicked()</signal>
   <receiver>kseNext</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clrPrev</sender>
   <signal>clicked()</signal>
   <receiver>ksePrev</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clrStop</sender>
   <signal>clicked()</signal>
   <receiver>kseStop</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clrAdd</sender>
   <signal>clicked()</signal>
   <receiver>kseAdd</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
